spec/Section 7 -- Response.md

```diff
@@ -244,6 +264,157 @@ discouraged.
 }
 ```
 
+### Incremental
+
+The `incremental` entry in the response is a non-empty list of Defer or Stream
+payloads. If the response of the GraphQL operation is a response stream, this
+field may appear on both the initial and subsequent values.
+
+For example, a query containing both defer and stream:
+
+```graphql example
+query {
+  person(id: "cGVvcGxlOjE=") {
+    ...HomeWorldFragment @defer(label: "homeWorldDefer")
+    name
+    films @stream(initialCount: 1, label: "filmsStream") {
+      title
+    }
+  }
+}
+fragment HomeWorldFragment on Person {
+  homeWorld {
+    name
+  }
+}
+```
+
+The response stream might look like:
+
+Response 1, the initial response does not contain any deferred or streamed
+results.
+
+```json example
+{
+  "data": {
+    "person": {
+      "name": "Luke Skywalker",
+      "films": [{ "title": "A New Hope" }]
+    }
+  },
+  "hasNext": true
```

Comment:
Should this message contain `incremental: []`? At the very least it might be helpful to have an example that shows non-empty `incremental` in the first message with an explanation.